app/design/frontend/UnitedArtsEducation/unitednow/Magento_Checkout/layout/checkout_index_index.xml

```diff
@@ -34,5 +34,8 @@
                 </argument>
             </arguments>
         </referenceBlock>
+        <referenceBlock name="top.search" remove="true" />
+        <referenceBlock name="minicart" remove="true" />
+        <referenceBlock name="header.promo" remove="true" />
```

Comment:
I think it would be best to remove "header.promo.outer", which includes the entire block, not just the contents.